communicating_vessels.py

```diff
@@ -0,0 +1,17 @@
+from collections.abc import Collection
+import itertools
+
+
+def interleave(*iterables: Collection) -> list:
+    """
+        The function receives one or more iterable parameters, and returns a list of intertwined items.
```

Comment:
I would an example to the documentation just so it will be more intuitive to the reader, but its great :)